lib/Config.js

```diff
@@ -30,16 +30,36 @@ class Config {
     }
 
     // default configuration
+    let dbhost = 'localhost';
+    let dbport = 3306;
+    let dbuser = 'xud';
+    let dbname = 'xud';
+    let dbpass = null;
+    if (process.env.DB_HOST !== '') {
+      dbhost = process.env.DB_HOST;
+    }
+    if (process.env.DB_PORT !== null) {
+      dbport = parseInt(process.env.DB_PORT, 10);
+    }
+    if (process.env.DB_USER !== '') {
+      dbuser = process.env.DB_USER;
+    }
+    if (process.env.DB_NAME !== '') {
+      dbname = process.env.DB_NAME;
+    }
+    if (process.env.DB_PASSWORD !== null) {
+      dbpass = process.env.DB_PASSWORD;
+    }
```

Comment:
I'm hesitant to use environment variables here. That's one more source of configuration to keep track of, plus as is this would get overwritten by command line and config file parameters. Instead of environment variables, could you maybe put any docker-specific configuration into a `xud.conf` file that is included in the container?

Reply:
I've refactored it load from xud.conf now.